### ISSUE TITLE
Print cert name in renewal errors

### ISSUE THREAD

**Issue by jsha** [certbot/certbot#4932]
Some errors during renewal print the path to a certificate, e.g.:

```
Attempting to renew cert from /etc/letsencrypt/renewal/jacob.hoffman-andrews.com-0001.conf produced an unexpected error: Account at /etc/letsencrypt/accounts/acme-staging.api.letsencrypt.org/directory/06c4ea2fe30100e185fea43f26817312 does not exist. Skipping.
```

It would be nice to also print the cert name, since that is the identifier used to delete certs through Certbot.

**Comment by r5d** [comment]
Does this look like the right way to do it?

```diff
diff --git a/certbot/renewal.py b/certbot/renewal.py
index 5c669794..d5d4b2ce 100644
--- a/certbot/renewal.py
+++ b/certbot/renewal.py
@@ -389,14 +389,17 @@ def handle_renewal_request(config):
         disp = zope.component.getUtility(interfaces.IDisplay)
         disp.notification("Processing " + renewal_file, pause=False)
         lineage_config = copy.deepcopy(config)
+        lineagename = storage.lineagename_for_filename(renewal_file)
 
         # Note that this modifies config (to add back the configuration
         # elements from within the renewal configuration file).
         try:
             renewal_candidate = _reconstitute(lineage_config, renewal_file)
         except Exception as e:  # pylint: disable=broad-except
-            logger.warning("Renewal configuration file %s produced an "
-                           "unexpected error: %s. Skipping.", renewal_file, e)
+            logger.warning("Renewal configuration file %s (cert: %s) "
+                           "produced an "
+                           "unexpected error: %s. Skipping.", lineagename,
+                               renewal_file, e)
             logger.debug("Traceback was:\n%s", traceback.format_exc())
             parse_failures.append(renewal_file)
             continue
@@ -422,8 +425,9 @@ def handle_renewal_request(config):
                     renew_skipped.append(renewal_candidate.fullchain)
         except Exception as e:  # pylint: disable=broad-except
             # obtain_cert (presumably) encountered an unanticipated problem.
-            logger.warning("Attempting to renew cert from %s produced an "
-                           "unexpected error: %s. Skipping.", renewal_file, e)
+            logger.warning("Attempting to renew cert (%s) from %s produced an "
+                           "unexpected error: %s. Skipping.", lineagename,
+                               renewal_file, e)
             logger.debug("Traceback was:\n%s", traceback.format_exc())
             renew_failures.append(renewal_candidate.fullchain)
 

```

**Comment by jsha** [comment]
I think so, roughly! Some line wrapping issues but looks generally reasonable.

**Comment by r5d** [comment]
@jsha, submitted PR  (#4957)  for review.

**Comment by bmw** [comment]
Fixed in #4957.
